crypto/poly1305/poly1305_ieee754.c

```diff
@@ -132,28 +132,28 @@ int poly1305_init(void *ctx, const unsigned char key[16])
 #if defined(__x86_64__)
         u32 mxcsr_orig;
 
-        asm volatile ("stmxcsr	%0":"=m"(mxcsr_orig));
-        asm volatile ("ldmxcsr	%0"::"m"(mxcsr));
+        asm volatile ("stmxcsr  %0":"=m"(mxcsr_orig));
+        asm volatile ("ldmxcsr  %0"::"m"(mxcsr));
```

Comment:
Even though it very likely works it's a bit ambiguous. There *are* assemblers that are picky about these things. You'll probably have hard time finding one, but trouble is that you won't check them all. But the least one knows that it does compile which tabs. With this in mind it might be appropriate to be more conservative here and at least let alone platforms that you don't explicitly test. Though on the other hand, this is non-production code, i.e. it's not actually compiled. So that if some assembler is found allergic, one can arguably address it then. But of course it would feel silly to change it one way, then back...

Reply:
> the least one knows that it does compile with tabs

I mean the least one knows it did compile the moment code was written.

Reply:
Ok, I will revert all changes inside asm strings.